docs/reference.asciidoc

```diff
@@ -2983,7 +2983,7 @@ client.searchableSnapshots.stats(...)
 ==== activate_user_profile
 Creates or updates the user profile on behalf of another user.
 
-https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-activate-user-profile.html[Endpoint documentation]
+https://www.elastic.co/guide/en/elasticsearch/reference/master/security-api-activate-user-profile.html[Endpoint documentation]
```

Comment:
Though this works, it might be better to lean on the shared attributes for future-proofing. For example:
```suggestion
{ref}/security-api-activate-user-profile.html[Endpoint documentation]
```